runtime/mainnet/src/config/xcm.rs

```diff
@@ -214,56 +244,746 @@ impl cumulus_pallet_xcm::Config for Runtime {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::Currency>(),
+				TypeId::of::<Balances>(),
+			);
+		}
+
+		#[test]
+		fn asset_matching_is_disabled() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::CurrencyMatcher>(),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn message_execute_xcm_origin_uses_signed_to_accountid32() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::ExecuteXcmOrigin>(),
+				TypeId::of::<
+					EnsureXcmOrigin<
+						RuntimeOrigin,
+						SignedToAccountId32<RuntimeOrigin, AccountId, RelayNetwork>,
+					>,
+				>(),
+			);
+		}
+
+		#[test]
+		fn limits_max_lockers() {
+			assert_eq!(<<Runtime as pallet_xcm::Config>::MaxLockers as Get<u32>>::get(), 8);
+		}
+
+		#[test]
+		fn max_remote_lock_consumers_is_0() {
+			assert_eq!(
+				<<Runtime as pallet_xcm::Config>::MaxRemoteLockConsumers as Get<u32>>::get(),
+				0
+			);
+		}
+
+		#[test]
+		fn remote_consider_identifier_is_disabled() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::RemoteLockConsumerIdentifier>(),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn send_xcm_origin_is_configured() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::SendXcmOrigin>(),
+				TypeId::of::<
+					EnsureXcmOrigin<
+						RuntimeOrigin,
+						SignedToAccountId32<RuntimeOrigin, AccountId, RelayNetwork>,
+					>,
+				>(),
+			);
+		}
+
+		#[test]
+		fn sovereign_account_uses_location_converter() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::SovereignAccountOf>(),
+				TypeId::of::<LocationToAccountId>(),
+			);
+		}
+
+		#[test]
+		fn does_not_have_trusted_lockers() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::TrustedLockers>(),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn universal_location_is_configured() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::UniversalLocation>(),
+				TypeId::of::<UniversalLocation>(),
+			);
+		}
+
+		#[test]
+		fn weigher_uses_fixed_weight_bounds() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::Weigher>(),
+				TypeId::of::<FixedWeightBounds<UnitWeightCost, RuntimeCall, MaxInstructions>>(),
+			);
+		}
+
+		#[test]
+		fn does_not_use_default_weights() {
+			assert_ne!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::WeightInfo>(),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn execute_filters_nothing() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::XcmExecuteFilter>(),
+				TypeId::of::<Everything>(),
+			);
+		}
+
+		#[test]
+		fn executor_is_configured() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::XcmExecutor>(),
+				TypeId::of::<XcmExecutor<XcmConfig>>(),
+			);
+		}
+
+		#[test]
+		fn reserve_transfer_filter_only_allows_dot_from_ah() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_xcm::Config>::XcmReserveTransferFilter>(),
+				TypeId::of::<Everything>(),
```

Comment:
name of test doesn't match what is occurring here.

Reply:
I see this was resolved here https://github.com/r0gue-io/pop-node/pull/459#pullrequestreview-2593873074